sympy/printing/ccode.py

```diff
@@ -299,6 +288,14 @@ def _print_Pow(self, expr):
             return '%spow%s(%s, %s)' % (self._ns, suffix, self._print(expr.base),
                                    self._print(expr.exp))
 
+    def _print_Mod(self, expr):
+        num, den = expr.args
+        if num.is_integer and den.is_integer:
+            template = "(({}) % ({}))"
+            return template.format(self._print(num), self._print(den))
+        elif not num.is_integer or not den.is_integer:
```

Comment:
I see this is just transcribed from my previous lambdas, but I think we can replace this with a simple `else:`?

Reply:
Sure.